source/client/renderer/LevelRenderer.cpp

```diff
@@ -811,17 +811,21 @@ void LevelRenderer::renderHitOutline(Player* pPlayer, const HitResult& hr, int i
 	glDisable(GL_TEXTURE_2D);
 	glDepthMask(false);
 
-	// Maximize Line Width
-	glEnable(GL_LINE_SMOOTH);
-	
+	// Determine Line Width
+	float line_width = 1.5f / Gui::InvGuiScale;
+	// Clamp Line Width
 	float range[2];
-	glGetFloatv(GL_SMOOTH_LINE_WIDTH_RANGE, range);
-
-	float lineWidth = 2.0f;
-	if (lineWidth > range[1])
-		lineWidth = range[1];
-
-	glLineWidth(lineWidth);
+	glGetFloatv(GL_ALIASED_LINE_WIDTH_RANGE, range);
+	if (range[1] < line_width)
+	{
+		line_width = range[1];
+	}
+	else if (range[0] > line_width)
+	{
+		line_width = range[0];
+	}
+	// Set Line Width
+	glLineWidth(line_width);
```

Comment:
Why switch to aliased lines?

Reply:
Because smoothed lines aren't actually supported on OpenGL ES 2.0. It was just failing silently.